opennlp-tools/src/main/java/opennlp/tools/tokenize/SimpleTokenizer.java

```diff
@@ -101,4 +107,12 @@ else if (Character.isDigit(c)) {
     }
     return tokens.toArray(new Span[tokens.size()]);
   }
+
+  private boolean isLineSeparator(char character) {
+    return character == Character.LINE_SEPARATOR || character == Character.LETTER_NUMBER;
+  }
+
+  public void setKeepNewLines(boolean keepNewLines) {
+    this.keepNewLines = keepNewLines;
+  }
```

Comment:
Would it make sense to move this common part to the parent abstract class? Can be done/discussed later too.

Reply:
Good call. I wrote [OPENNLP-1364](https://issues.apache.org/jira/browse/OPENNLP-1364) to capture this.